### PR TITLE
Build etcd container image for kubic project

### PR DESCRIPTION
**Pull request by mjura**

### CODE REVIEW

**Comment by sysrich** [comment]
Also, please make sure the resulting image is submitted to devel:kubic:containers - Kubic no longer uses containers from devel:CaaSP

**Comment by davidcassany** [comment]
> Kubic no longer uses containers from devel:CaaSP

Oh, then I guess the automation to commit github updates to OBS should be also updated to use `devel:kubic:containers`.

@sysrich shall the `devel:CaaSP:kubic-containers` be removed then? I am wondering if it makes any sense having CaaSP specific containers build against Factory in OBS. My understanding is that CaaSP containers are a subset of Kubic containers (probably there are also some that are CaaSP only and not present in Kubic) but building on base of SLE.

Since we are not building containers against SLE in OBS (too many packages missing, last time I tried it was frustrating) I can't see the value of having the `devel:CaaSP:kubic-containers` project.

Does it make any sense?

File: etcd-image/etcd-image.kiwi.ini [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      <containerconfig
+        name="_PRODUCT_/etcd"
+        tag="%%SHORT_VERSION%%"
+        maintainer="SUSE Containers Team &lt;containers@suse.com&gt;">
+        <expose>
+          <port number="2379"/>
+          <port number="2380"/>
+        </expose>
+        <entrypoint execute="bash">
+            <argument name="/usr/local/bin/entrypoint.sh"/>
```
[the remark this answers is not in the document]
Comment (mjura) [reply etcd-image/etcd-image.kiwi.ini]: I've just added this

**Comment by davidcassany** [comment]
@mjura, do we have a clear motivation not to include a feature like what [migrate-if-needed.sh](https://github.com/kubernetes/kubernetes/blob/master/cluster/images/etcd/migrate-if-needed.sh) is doing? To my understanding in an environments like Tumbleweed (rolliing release) this is an important feature. However I don't know how this image is expected to work in kubic.

**Comment by mjura** [comment]
@davidcassany: Script `migrate-if-needed.sh` is used for migration between etcd backend version v2 to v3. We are already building etcd-image with etcd version 3.3.10, where default backend is already v3. So script is not needed for us.

**Pull request merged**